src/quo2/components/settings/settings_list/view.cljs

```diff
@@ -0,0 +1,133 @@
+   {:flex 1}
+   [text/text
+    {:accessibility-label :setting-item-name-text
+     :ellipsize-mode      :tail
+     :style               (style/title)
+     :number-of-lines     1
+     :weight              :medium
+     :size                :paragraph-1}
+    title]
+  ])
+
+(def browser-context-icon
+  [rn/view
+   [quo2.icons/icon :browser-context
+    {:container-style style/icon
+     :color           (colors/theme-colors
+                       colors/neutral-50
+                       colors/neutral-40)}]])
+
+(def chevron-icon
+  [rn/view
+   [quo2.icons/icon :chevron-right
+    {:color (colors/theme-colors
+             colors/neutral-50
+             colors/neutral-40)}]])
+
+(defn switch-button
+  [{:keys [:checked?
+           on-change]}]
+  [selectors/toggle
+   {:checked?             checked?
+    :on-change            (fn [new-value] (on-change new-value))
+    ;; TODO: For some reason this prop is not working on ios simulator, spike a little bit more
+    :ios-background-color (colors/theme-colors
+                           colors/neutral-30
+                           colors/neutral-80)
+    ;; TODO: Ask for colors on the track, there is no definition on the design
+    ;; Colors for false value only work on Android
+    ;; :track-color          {:false (colors/theme-colors
+    ;;                               colors/danger
+    ;;                               colors/danger)
+    ;;                       :true  (colors/theme-colors
+    ;;                               colors/danger
+    ;;                               colors/danger)}
+    :style                style/switch-button}]
+)
+
+(def badge-icon
+  [rn/view
+   {:style (style/dot)}])
+
+(defn right-button
+  [{:keys [title
+           on-press]}]
+  [button/button
+   {:type     :outline
+    :on-press on-press
+    :size     24}
+   title])
+
+(defn communities-icons
+  [{:keys [data
+           icon-style]}]
+  (let [communities-count (dec (count data))]
+    [rn/view
+     {:style style/communities-container}
+     (map-indexed
+      (fn [index {:keys [source accessibility-label]}]
+        [rn/image
+         {:key                 source
+          :source              (if (string? source)
+                                 {:uri source}
+                                 source)
+          :accessibility-label accessibility-label
+          :style               (merge (style/community-icon (- communities-count index)) icon-style)}])
+      data)]))
+
+(defn settings-list
+  "Options
+   - `title` String to show in the center of the component, right to the icon and left to optional gadgets.
+   - `on-press` Callback called when the component is pressed.
+   - `accessibility-label` String to use as accessibility-label for VoiceOver.
+   - `left-icon` Symbol to indicate icon type on the left side of the component.
+   - `chevron?` Boolean to show/hide chevron at the right border of the component.
+   - `switch-prop` Object with the following properties:
```

Comment:
I'd recommend not calling this an object with properties, since it's just a clojure map.

Reply:
Ok, will change Object => Map and properties => keys, is that ok?